src/sources/kubernetes_logs/mod.rs

```diff
@@ -236,7 +237,6 @@ struct Source {
     fingerprint_lines: usize,
     glob_minimum_cooldown: Duration,
     ingestion_timestamp_field: Option<String>,
-    timezone: TimeZone,
```

Comment:
It's still on the actual configuration type (`Config`). It's removed here because it wasn't actually ever used from what I could tell, given the format of the datetime field in the Kubernetes runtime logs being RFC3339 and already having the timezone information.